kiali-server/values.yaml

```diff
@@ -26,6 +26,9 @@ auth:
   strategy: ""
 
 deployment:
+  # This is a user defined configmap that should hold the configuration for Kiali. It will be used instead of the default one if set.
+  # The custom configmap should have a data entry for "config.yaml", which will be used by Kiali at startup.
+  configuration_configmap: ""
```

Comment:
This should not go under deployment. This is an external configuration setting (specific to the server helm chart itself). These settings under "deployment" mimic those that are in the Kiali CR. This should be a top-level value like `fullnameOverride`.

Reply:
This request-for-change still stands and should be done in conjunction with my other suggested change in the new review-for-change I just submitted (i.e. move this to the top level and then have the helm chart read in this external configmap and store the values in .Values so the helm charts can use the values in the external configmap as its defaults).